apps/meteor/app/file-upload/server/methods/sendFileMessage.ts

```diff
@@ -30,121 +31,127 @@ function validateFileRequiredFields(file: Partial<IUpload>): asserts file is AtL
 }
 
 export const parseFileIntoMessageAttachments = async (
```

Comment:
I'd suggest creating another function named `parseFilesIntoMessageAttachments` which receives an array of files (please name if `files`, not `filearr`), calls this method for each file and merge the results to return. That way places that are using this function doesn't need to be updated and you can remove the hack of checking if the filearr is an array.

Reply:
@rodrigok sir I've added a new function called `parseMultipleFilesIntoMessageAttachments`. Currently, it's located in the same file as the component I'm working on. Could you advise on where this function should be moved to align with our project's structure?